### ISSUE TITLE
Schema & Docs: item descriptions need to be added for InterestType

### ISSUE THREAD

**Issue by siwhitehouse** [openownership/data-standard#178]
This is a child issue of #160
Currently, there are no descriptions in the InterestType codelist.

code | title | description | technical note
-- | -- | -- | --
shareholding | Shareholding |   |  
voting-rights | Voting rights |   |  
appointment-of-board | Appointment of board |   |  
influence-or-control | Influence or control |   |  
senior-managing-official | Senior managing official |   |  
settlor-of-trust | Settlor of trust |   |  
trustee-of-trust | Trustee of a trust. |   |  
protector-of-trust | Protector of a trust |   |  
beneficiary-of-trust | Beneficiary of a trust |   |  
other-influence-or-control-of-trust | Other influence or control of a trust |   |  
rights-to-surplus-assets | Rights to surplus assets |   |  
rights-to-profit-or-income | Rights to receive profits or income |   |  

Below are my first attempts at providing descriptions

code | title | description | technical note
-- | -- | -- | --
shareholding | Shareholding | The shares in a company that a particular person or organisation owns considered together as a unit. | [source](https://dictionary.cambridge.org/dictionary/english/shareholding) 
voting-rights | Voting rights | The right of shareholders to vote on matters of corporate policy, including decisions on the makeup of the board of directors, issuing securities, initiating corporate actions and making substantial changes in the corporation's operations. | [source](https://www.investopedia.com/terms/v/votingright.asp) 
appointment-of-board | Appointment of board | The absolute right to appoint members of the board of directors  |  
influence-or-control | Influence or control |   |  
senior-managing-official | Senior managing official | The person who exercises control over the management of the entity. In the absence of a natural person(s) that exercises control of the Entity through ownership interests, the Senior Managing Official can be identified as the Controlling Person(s) of the Entity. | [source](https://www.crs.hsbc.com/en/glossary)
settlor-of-trust | Settlor of trust | A person who settles property on trust law for the benefit of beneficiaries. In some legal systems, a settlor is also referred to as a trustor, or occasionally, a grantor or donor. |  [source](https://en.wikipedia.org/wiki/Settlor)
trustee-of-trust | Trustee of a trust. | A person or firm that holds and administers property or assets for the benefit of a third party  |  [source](https://www.investopedia.com/terms/t/trustee.asp)
protector-of-trust | Protector of a trust | A person appointed under the trust instrument to direct or restrain the trustees in relation to their administration of the trust.  |  [source](https://en.wikipedia.org/wiki/Protector_(trust))
beneficiary-of-trust | Beneficiary of a trust | The person or legal entity who profits from a trust's holdings or activities.  |  From [glossary](https://docs.google.com/spreadsheets/d/1SfjpIkf4GmdHPSo2d9dzInte2TNdOx1vHfKmq5UoXAc/edit?pli=1#gid=0)
other-influence-or-control-of-trust | Other influence or control of a trust | Any influence or control in a trust that is distinct from being a settlor, trustee, protector or beneficiary.  |  
rights-to-surplus-assets | Rights to surplus assets | The right to a share in the amount of an asset or resource that exceeds the portion that is utilized. A surplus is used to describe many excess assets including income, profits, capital, and goods.  |  [adapted from](https://www.investopedia.com/terms/s/surplus.asp) 
rights-to-profit-or-income | Rights to receive profits or income | The right to a share in the profits or income of an entity |  

Note that I have misused the "technical note" column to show my sources. If no source is referenced then I have written the description myself. I have left a couple of descriptions blank where I don't feel confident to make a start.

I've written "appointment-to-the-board" as an absolute right as this distinguishes it, in my mind, from voting rights shared by multiple share holders.

Some of the above can be expressed by percentages (shareholdings for eg.) whereas others are boolean properties (senior managing official). Looking at the [Interest object](http://standard.openownership.org/en/v0-1/schema/reference.html?highlight=codelist#interest) it has a property for share (percentage share) which I don't think applies to all of the codes in the list. (This may well be a tangent for this issue, which is to complete the descriptions).

**Comment by kd-ods** [comment]
Just noting here that in PR #184 the interestType codelist has been amended:
 - 'rights-to-surplus-assets' is renamed 'rights-to-surplus-assets-on-dissolution'
 - 'rights-granted-by-contract' has been added
 - 'conditional-rights-granted-by-contract' has been added

**Comment by siwhitehouse** [comment]
Thanks @kd-ods - in which case I'll suggest the following amendment

code | title | description | technical note
-- | -- | -- | --
rights-to-surplus-assets-on-dissolution | Rights to surplus assets | The right to a share in the amount of an asset or resource that exceeds the portion that is utilized upon the winding up of an entity. | [adapted from Investopedia](https://www.investopedia.com/terms/s/surplus.asp) and[ UK PSC](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/675103/Statutory_guidance_LLPs.pdf) 

and the following additional codelist entries

code | title | description | technical note
-- | -- | -- | --
rights-granted-by-contract' | Rights granted by contract | A contract that defines beneficial ownership rights beyond those otherwise implied by ownership structures. |
conditional-rights-granted-by-contract | Conditional rights granted by contract | An agreement that defines conditions by which beneficial ownership rights could come into existence | 

I don't follow the logic of adding the last two codelist entries. All of the other entries, with the possible exception of 'shareholding', describe the way in which an interest can be exercised, but these last two describe the way in which the interest is gained.

**Comment by kd-ods** [comment]
I'd suggest rewording all the descriptions along the lines of...

shareholding:
An economic interest in an entity gained by holding shares.

rights-granted-by-contract:
A controlling or economic interest in an entity granted via a contractual arrangement.

**Comment by siwhitehouse** [comment]
code|title|description|technical note|
-- | -- | -- | --
shareholding|Shareholding|"The shares in a company that a particular person or organisation owns considered together as a unit."||
voting-rights|Voting rights|"The right of shareholders to vote on matters of corporate policy| including decisions on the makeup of the board of directors| issuing securities| initiating corporate actions and making substantial changes in the corporation's operations."||
appointment-of-board|Appointment of board|"The absolute right to appoint members of the board of directors."||
influence-or-control|Influence or control
senior-managing-official|Senior managing official|"The person who exercises control over the management of the entity. In the absence of a natural person(s) that exercises control of the Entity through ownership interests| the Senior Managing Official can be identified as the Controlling Person(s) of the Entity."||
settlor-of-trust|Settlor of trust|"A person who settles property on trust law for the benefit of beneficiaries. In some legal systems| a settlor is also referred to as a trustor| or occasionally| a grantor or donor."||
trustee-of-trust|Trustee of a trust.|"A person or firm that holds and administers property or assets for the benefit of a third party."||
protector-of-trust|Protector of a trust|"A person appointed under the trust instrument to direct or restrain the trustees in relation to their administration of the trust."||
beneficiary-of-trust|Beneficiary of a trust|"The person or legal entity who profits from a trust's holdings or activities."||
other-influence-or-control-of-trust|Other influence or control of a trust|"Any influence or control in a trust that is distinct from being a settlor| trustee| protector or beneficiary."||
rights-to-surplus-assets-on-dissolution|Rights to surplus assets on dissolution|"The right to a share in the amount of an asset or resource that exceeds the portion that is utilized upon the winding up of an entity."||
rights-to-profit-or-income|Rights to receive profits or income|"Beneficial ownership rights beyond those otherwise implied by ownership structures that are granted by contract."||
rights-granted-by-contract|Rights granted by contract|"Conditional beneficial ownership rights beyond those otherwise implied by ownership structures that are granted by contract."||

**Comment by siwhitehouse** [comment]
[All codelists have been edited](https://github.com/openownership/data-standard/tree/0.2-dev-codelists-160/schema/codelists) with this one changed as per the discussion in this issue.

**Comment by ScatteredInk** [comment]
> I don't follow the logic of adding the last two codelist entries. All of the other entries, with the possible exception of 'shareholding', describe the way in which an interest can be exercised, but these last two describe the way in which the interest is gained.

@siwhitehouse I agree that this list of interest types is becoming slightly crowded/confused. The other interest type that is fuzzy/forward-looking is `beneficiary-of-trust`. This can be a simple named individual but in a discretionary trust or other complex structure, it could a class of individuals ("the children of X") or a person mentioned in the deeds with some condition attached ("when X is 21 and has solved a series of complex riddles").

I think in 0.3 we might want to look at the original conceptual model in #10 again, and have a think about roles vs interests, as this list would be relatively easy to split out into a more coherent pattern.

**Comment by ScatteredInk** [comment]
Done in #194
